springboard/views.py

```diff
@@ -20,14 +20,23 @@ def __init__(self, request):
         self.language = request.locale_name
         self.settings = request.registry.settings
 
-        repo_name = parse_repo_name(self.settings['unicore.content_repo_url'])
-        repo_path = os.path.join(
-            self.settings.get('unicore.repos_dir', 'repos'), repo_name)
-        index_prefix = slugify(repo_name)
-        self.workspace = EG.workspace(
-            repo_path, index_prefix=index_prefix)
-        self.all_categories = self.workspace.S(Category)
-        self.all_pages = self.workspace.S(Page)
+        repo_dir = self.settings.get('unicore.repos_dir', 'repos')
+        repo_names = map(
+            lambda repo_url: parse_repo_name(repo_url),
+            self.settings['unicore.content_repo_urls'].strip().split('\n'))
+        self.all_repo_paths = map(
+            lambda repo_name: os.path.join(repo_dir, repo_name),
+            repo_names)
+        self.all_index_prefixes = map(
+            lambda repo_name: slugify(repo_name),
+            repo_names)
+
+        search_config = {
+            'in_': self.all_repo_paths,
+            'index_prefixes': self.all_index_prefixes
+        }
+        self.all_categories = SM(Category, **search_config)
+        self.all_pages = SM(Page, **search_config)
```

Comment:
This may have slipped my attention earlier when reviewing these changes in Elastic-Git but how does an `SM` instance know which elasticsearch to connect to?